src/app/profile/profile.component.html

```diff
@@ -0,0 +1,37 @@
+<form [formGroup]="profileForm" (ngSubmit)="onSubmit(profileForm.value)">
+    <h2>Profile</h2>
+    <p>
+        <label for="firstName">First Name</label>
+        <input type="text" id="firstName" formControlName="firstName">
+        <span class="error-message" *ngIf="firstName.hasError('required') && firstName.dirty">Required Field</span>
+    </p>
+    <p>
+        <label for="lastName">Last Name</label>
+        <input type="text" id="lastName" formControlName="lastName">
+        <span class="error-message" *ngIf="lastName.hasError('required') && lastName.dirty">Required Field</span>
+    </p>
+    <p>
+        <label for="email">Email</label>
+        <input type="text" id="email" formControlName="email">
+        <span class="error-message" *ngIf="emailError()">Must include "@" symbol</span>
+    </p>
+    <hr>
+    <h2>Phone Numbers</h2>
+    <div formArrayName="phoneNumbers" *ngFor="let numbers of PhoneNumbersArray.controls; let i = index">
```

Comment:
ISSUE: I had trouble getting this ngFor to work. A lot of the tutorials wanted me to have the getter code (similar to this.profileForm.get('phoneNumbers')) here in the template instead of PhoneNumbersArray.controls which references the code in the component.  Is what I did correct?

Reply:
I prefer reducing the amount of js code in the template.   Your way is also providing some utility in your other function calls inside the component which is nice.

I wasn't aware you could use the index value 'i' as the parameter into [formGroupName] when working with a FormArray like that.  Ya learn something new every day. :)  I normally use [formGroup] when I am binding.

In this approach your "numbers" variable doesn't get used.    In this case, I would rename "numbers" to "_", also known as a discard.  This will communicate to other engineers that the value is not used.  Or if it were used, since PhoneNumbersArray.controls is technically an Array of FormGroup, I'd name it "phoneGroup" to better describe what it is.  I would also rename "i" to "phoneGroupIndex" to clarify its meaning.

You can print out numbers like you did with the other form groups.
<pre>{{numbers.value | json}}</pre>

Reply:
RE issue #2 - Gotcha. I kept noticing in my research on this that people were using '_' . I didnt understand what that syntax meant, so i just gave it a random name. Thanks for the info! I will make this change in the code for my future reference.